sdks/cpp/connections/gRPC/examples/status_update/device.status_update.json

```diff
@@ -32,13 +32,26 @@
   "params": {
     "counter": {
       "type": "INT32",
-      "name": {"display_strings": {"en": "Counter", "es": "Contador", "fr": "Compteur"}},
-      "value": { "int32_value": 0 }
+      "name": {
+        "display_strings": {
+          "en": "Counter",
+          "es": "Contador",
+          "fr": "Compteur"
+        }
+      },
+      "value": { "int32_value": 0 },
+      "constraint": { "ref_oid": "/constraints/counter_cap" }
     },
     "hello": {
       "type": "STRING",
-      "name": {"display_strings": {"en": "Text Box", "es": "Caja de texto", "fr": "Boîte de texte"}},
```

Comment:
please leave this as a single line - we've had "sensitivity" to too much whitespace.

Reply:
I see, I might have to tweak my prettier config from the default